.github/workflows/publish-and-release.yml

```diff
@@ -130,39 +130,39 @@ jobs:
         rust_version: [stable]
         experimental: [false]
         include:
-        - os: macos-latest
-          artifact_name: c2patool_mac_universal.zip
-          uploaded_asset_name: c2patool_mac_universal-${{ needs.repo-prep.outputs.new-tag }}.zip
-        - os: ubuntu-latest
-          artifact_name: c2patool_linux_intel.tar.gz
-          uploaded_asset_name: c2patool_linux_intel-${{ needs.repo-prep.outputs.new-tag }}.tar.gz
-        - os: windows-latest
-          artifact_name: c2patool_win_intel.zip
-          uploaded_asset_name: c2patool_win_intel-${{ needs.repo-prep.outputs.new-tag }}.zip
+          - os: macos-latest
+            artifact_name: c2patool_mac_universal.zip
+            uploaded_asset_name: c2patool-${{ needs.repo-prep.outputs.new-tag }}-universal-apple-darwin.zip
+          - os: ubuntu-latest
+            artifact_name: c2patool_linux_intel.tar.gz
+            uploaded_asset_name: c2patool-${{ needs.repo-prep.outputs.new-tag }}-x86_64-unknown-linux-gnu.tar.gz
+          - os: windows-latest
+            artifact_name: c2patool_win_intel.zip
+            uploaded_asset_name: c2patool-${{ needs.repo-prep.outputs.new-tag }}-x86_64-pc-windows-msvc.zip
```

Comment:
I've changed these names to more closely match what `cargo binstall` [looks for](https://github.com/cargo-bins/cargo-binstall/blob/main/SUPPORT.md).